src/tbb/governor.cpp

```diff
@@ -51,6 +51,10 @@ namespace rml {
 tbb_server* make_private_server( tbb_client& client );
 } // namespace rml
 
+namespace system_topology {
+    void destroy();
```

Comment:
In governor you use system in tbb_bind platform, so please consider use one of them.
```suggestion
    void destroy_platform_topology();
```